### PR TITLE
GHA build docker image and publish to Packages

### PR DESCRIPTION
**Pull request by derekadombek**
# Pull Request

## Description
This is an action for automating our deployment of our OCR API

## Related Issues
none

## Additional Notes
[Add any additional context or notes that reviewers should know about.]

## Checklist
Please review and complete the following checklist before submitting your pull request:

- [x] I have ensured that the pull request is of a manageable size, allowing it to be reviewed within a single session.
- [x] I have reviewed my changes to ensure they are clear, concise, and well-documented.
- [x] I have updated the documentation, if applicable.
- [x] I have added or updated test cases to cover my changes, if applicable.
- [x] I have minimized the number of reviewers to include only those essential for the review.
- [x] I have notified teammates in the review thread to build awareness.

## Checklist for Reviewers
Please review and complete the following checklist during the review process:

- [ ] The code follows best practices and conventions.
- [ ] The changes implement the desired functionality or fix the reported issue.
- [ ] The tests cover the new changes and pass successfully.
- [ ] Any potential edge cases or error scenarios have been considered.

### CODE REVIEW

**Review by zdeveloper — APPROVED** [review]

File: .github/workflows/build-ocr-docker.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        uses: docker/login-action@v3
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Extract metadata (tags, labels) for Docker
+        id: meta
+        uses: docker/metadata-action@v5
+        with:
+          images: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}-ocr-api
```
Comment (derekadombek) [inline .github/workflows/build-ocr-docker.yml:36]: @zdeveloper I made this change after you approved. can i get another ✅

**Review by zdeveloper — APPROVED** [review]
